apps/learner/src/app.css

```diff
@@ -1,6 +1,12 @@
 @import url('https://fonts.googleapis.com/css2?family=Geist:wght@100..900&display=swap');
 @import 'tailwindcss';
 
+@layer base {
+  * {
+    scrollbar-color: color-mix(in oklab, var(--color-slate-950) 20%, transparent) transparent;
```

Comment:
Just curious, is there a specific reason why a subtle or faint scrollbar is preferred over a darker, more visible one?

Reply:
No reason. We can use darker if we like.